components/eam/src/physics/crm/physpkg.F90

```diff
@@ -744,25 +768,25 @@ subroutine phys_run1(phys_state, ztodt, phys_tend, pbuf2d,  cam_in, cam_out)
 #endif
 
   !-----------------------------------------------------------------------------
-  ! Tendency physics before flux coupler invocation
+  ! Physics tendency before coupler - Phase 1
   !-----------------------------------------------------------------------------
 
   call t_barrierf('sync_bc_physics', mpicom)
   call t_startf ('bc_physics')
+  call t_startf ('bc_physics1')
 
 !$OMP PARALLEL DO PRIVATE (C, beg_count, phys_buffer_chunk, end_count, chunk_cost)
   do c=begchunk, endchunk
 
     beg_count = shr_sys_irtc(irtc_rate)
 
-    ! Output physics terms to IC file
```

Comment:
Does this removed comment actually belong with the line below, the `diag_physvar_ic` call?

Reply:
yea, I'll add it back

Reply:
done.